spec/bookshelf-spec.ts

```diff
@@ -1520,11 +1528,11 @@ describe('Bookshelf relations', () => {
   });
 
   it('should give an option to sepcify relation types with a function', () => {
```

Comment:
Let's update the description of this test: 'should specify an option to format the type with a function'